.travis.yml

```diff
@@ -1,21 +1,26 @@
 language: python
 matrix:
-    include:
-        - python: 2.7
-          env: TOXENV=py27
-        - python: 3.4
-          env: TOXENV=py34
-        - python: 3.5
-          env: TOXENV=py35
-        - python: 3.6
-          env: TOXENV=py36
-        - python: 3.6
-          env: TOXENV=docs
-        - python: 3.6
-          env: TOXENV=flake8
-        - python: 3.6
-          env: TOXENV=pre-commit
+  include:
+  - python: 2.7
+    env: TOXENV=py27
+  - python: 3.4
+    env: TOXENV=py34
+  - python: 3.5
+    env: TOXENV=py35
+  - python: 3.6
+    env: TOXENV=py36
+  - python: 3.6
+    env: TOXENV=docs
+  - python: 3.6
+    env: TOXENV=flake8
+  - python: 3.6
+    env: TOXENV=pre-commit
 install: pip install tox coveralls
 script: tox
 after_success: coveralls
 sudo: false
+deploy:
+  provider: pypi
+  user: "yelplabs"
+  password:
+    secure: CRHU+aWPqlHKrVu+R8aMc0usIKy4mWiHtt8MCADU1oC/Z2SOB+jtVBEHP30Ze9cr1GLzfElAYDdNqI4oe2vUvvG103qJIAB4l8Cz/B40ZHVkHdsTU0rdacOPQ3bD5aGf3GqcjDcKKdk6AwohXNJ5mJzADiHB3kIb6CtS+IgQ0T1+bCFb5VX02ygpQF6+0Qywtw8EbVPZZokbQC0IEF30dtmmVAK7tF3CJcExgNWH2IvRyws7NZJLID5pcVDr+GAIGtyRsRy9l7JLCJ1B14Lliya/IBLzLN2uAE/rpD0ZZjw9y30AwMIqpcN8rqEMb0HFgIDqTFG9fc4g2ztyh+rwqIlWDKAZrvKfYnIQRqwHQkTx9VbxCSSZepx1NFHV7x4NF99xQqWMd7Lh1/PxmsAogZwpoVtyDxPmCT3hO+fs2XF4Ubbt//hr3yrVigW/Xa9R8k6a/dqzn7AGEgZBngGwdK9wo4cOsq67iiZbQCXnnZs2eaLmA9UR1pB8f90WyqidMnQTdPRMuRtzgvpXtZsJDi9+KHK/8idJMgxmNrF0QD2I7ZzFAPG2FO7lQNj1Qlx2PS2mU90jISlnYU1duda3IckD3iUs/8Z3n6kO0LAOq2/ci1Y65Vu1k/Cht+aOolP4WQ4mRJ0mqHGAmWGRFs0iXE8m1aO+WtU46q2BVs6dxNQ=
```

Comment:
you're missing the `on` condition and `distributions`. See https://github.com/Yelp/py_zipkin/blob/master/.travis.yml

Without those travis will error on every non-tagged commit and won't upload wheels

Reply:
Fair. I was really just trying to get the password bit submitted. I have no idea how you want this uploading configured.